website/docs/providers/terraform/d/remote_state.html.md

```diff
@@ -6,18 +6,87 @@ description: |-
   Accesses state meta data from a remote backend.
```

Comment:
```suggestion
  Accesses state metadata from a remote backend.
```
I know this isn't part of your PR, but perhaps you could clean this up while you're in here?

Reply:
Oh, indeed! I must admit I didn't really pay much attention to the header information while I was working on this, but since you pointed it out I rewrote the description entirely because I'm not really sure what "state metadata" was intended to mean here, and it seems unnecessarily vague for a data source that actually has a very specific and narrow scope. :man_shrugging: